plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/inspector-controls.tsx

```diff
@@ -63,6 +64,10 @@ interface WithInspectorControlsRequiredProps< T > {
 	attributes: InspectorControlsRequiredAttributes &
 		EditorBlock< T >[ 'attributes' ];
 	setAttributes: InspectorControlsProps[ 'setAttributes' ];
+	isBgVisible: {
+		isBackgroundVisible: boolean;
+		message?: string | null;
```

Comment:
Let's make it required

```suggestion
		message: string | null;
```

Otherwise, `message` could be `string | null | undefined` and create not necessary complexity